homework/index.js

```diff
@@ -30,14 +30,93 @@
     return elem;
   }
 
+  function createOption(repositories, select) {
+    createAndAppend('option', select, { text: 'Select a repository', disabled: 'disabled' });
+    repositories
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .forEach(repository => {
+        createAndAppend('option', select, { text: repository.name, value: repository.name });
+      });
+  }
+
+  function renderRepoBasisInfo(repository, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repository}`;
+    const container = document.getElementById('root');
+    fetchJSON(url, (err, repo) => {
+      if (err) {
+        createAndAppend('div', container, { text: err.message, class: 'alert-error' });
+      }
+      ul.innerHTML = '';
+
+      const li = createAndAppend('li', ul, { class: 'info-list' });
+      const table = createAndAppend('table', li);
+      const tbody = createAndAppend('tbody', table);
+      const trName = createAndAppend('tr', tbody);
+      createAndAppend('th', trName, { text: 'Repository:' });
+      const tdName = createAndAppend('td', trName);
+      createAndAppend('a', tdName, { text: repo.name, href: repo.html_url, target: '_blank' });
+      const trDes = createAndAppend('tr', tbody);
+      createAndAppend('th', trDes, { text: 'Description:' });
+      createAndAppend('td', trDes, { text: repo.description });
+      const trForks = createAndAppend('tr', tbody);
+      createAndAppend('th', trForks, { text: 'Forks:' });
+      createAndAppend('td', trForks, { text: repo.forks_count });
+      const trUpd = createAndAppend('tr', tbody);
+      createAndAppend('th', trUpd, { text: 'Updated at:' });
+      createAndAppend('td', trUpd, { text: repo.updated_at });
```

Comment:
Can you display this date in a more user-friendly format? Hint: create a JavaScript `Date` object from the date string and use one of its string methods to format the date.